Week3/homework/step4.js

```diff
@@ -2,7 +2,17 @@
 
 // Add your code here
 
+function createBase(value) {
+
+    const addNum = function (num) {
+        return value + num;
+    };
+    return addNum;
+}
+
 const addSix = createBase(6);
 
-addSix(10); // returns 16
-addSix(21); // returns 27
+const result1 = addSix(10);
+console.log(result1); // returns 16
+const result2 = addSix(21);
+console.log(result2);// returns 27
```

Comment:
👍 And I like that you created a named internal function, it makes it much easier to read. 😄